include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -2953,12 +2953,11 @@ __pattern_remove_if(__parallel_tag<_IsVector> __tag, _ExecutionPolicy&& __exec,
 //------------------------------------------------------------------------
 // merge
 //------------------------------------------------------------------------
-// Serial version of ___merge_path_out_lim merges the 1st sequence and the 2nd sequence in "reverse order":
-// the identical elements from the 2nd sequence are merged first.
-template <typename _Iterator1, typename _Iterator2, typename _Iterator3, typename _Comp>
+template <typename _Iterator1, typename _Iterator2, typename _Iterator3, typename _Comp,
+          typename _Proj1 = oneapi::dpl::identity, typename _Proj2 = oneapi::dpl::identity>
```

Comment:
```suggestion
template <typename _Iterator1, typename _Iterator2, typename _Iterator3, typename _Comp, typename _Proj1, typename _Proj2>
```
- we specify `_Proj1` and `_Proj2` in all calls so I think this defaults not required here.